extract-metadata.py

```diff
@@ -7,16 +7,29 @@
 import xml
 import xml.sax
 import yaml
-
+import toml
 
 def extract_md(filename):
     try:
         with open(filename, "r") as fh:
-            return next(yaml.safe_load_all(fh))
+            section_header = fh.readline().strip()
+            if (section_header == r'%%%'):
```

Comment:
```suggestion
            if section_header == r"%%%":
```

Would you mind running [black](https://pypi.org/project/black/) over this script?